iina/config/iina-default-input.conf

```diff
@@ -23,7 +23,7 @@ Shift+RIGHT  sub-seek  1
 Meta+s screenshot
 
 Meta+l ab-loop
-Meta+L cycle-values loop "inf" "no"
+Alt+Meta+l cycle-values loop "inf" "no"
```

Comment:
As the log viewer is the new feature I would think it would be better to pick a key binding that doesn't conflict with an existing binding.

Why does the log view binding not appear in the key bindings file like we do for the keys that bring up other panels?